config/examples/vmagent.yaml

```diff
@@ -12,6 +12,7 @@ spec:
   staticScrapeSelector: {}
   staticScrapeNamespaceSelector: {}
   replicaCount: 1
+  serviceAccountName: vmagent
```

Comment:
lets leave it empty. In this case operator will create needed role itself. It must simply deployment for vmagent.